parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ParquetMetadata.java

```diff
@@ -84,6 +86,9 @@ public static ParquetMetadata fromJSON(String json) {
   private final FileMetaData fileMetaData;
   private final List<BlockMetaData> blocks;
 
+  @JsonIgnore
```

Comment:
Why is this annotation required?

Reply:
The jackson mapper will not serialize this field to json with this annotation and keep the the same behavior as before.